composer.json

```diff
@@ -14,7 +14,7 @@
         "symfony/framework-bundle" : "~3.0"
     },
     "require-dev" : {
-        "atoum/atoum" : "master-dev",
+        "atoum/atoum" : "dev-master",
```

Comment:
As atoum is now fully compliant with PHP 7.x, we should have to explicitly use `~3.0`